src/js/visual/configurator-prc.js

```diff
@@ -1028,6 +1030,8 @@ ripe.ConfiguratorPrc.prototype._updateConfig = async function(animate) {
     // nature of execution) returns the control flow immediately
     if (!this.element) return;
 
+    this.useMasks = this.owner.brand && this.owner.hasTag("no_masks") ? false : this.useMasks;

```

Comment:
Why do you need to do this twice?

Reply:
In case the model changes and therefore the tags might also change

Reply:
Sure but wouldn't it suffice to do it here? Isn't `_updateConfig` called the first time?

Reply:
Yes, but for example in the case of ripe-white, the updateOptions sets it to true as default, and the tags should take precedence, I think